wpimath/src/main/java/edu/wpi/first/math/apriltag/AprilTagFieldLayout.java

```diff
@@ -0,0 +1,34 @@
+package edu.wpi.first.math.apriltag;
+
+import edu.wpi.first.math.geometry.Pose3d;
+import edu.wpi.first.math.geometry.Rotation3d;
+import edu.wpi.first.math.geometry.Translation3d;
+
+import java.util.HashMap;
+import java.util.Map;
+
+public class AprilTagFieldLayout {
+    private final Map<Integer, Pose3d> m_tags = new HashMap<>();
```

Comment:
Why are using a map instead of an array? Are we not expecting gaps in ids?

Reply:
It's not impossible.  I don't believe there will be on the official field, but if a team decides to make a layout for home use, it is possible they choose tags 3, 5, and 9 for example to make their layout with.